raw/server.js

```diff
@@ -0,0 +1,116 @@
+'use strict';
+
+if (process.env.NODE_ENV === 'production') {
+  require('@google-cloud/debug-agent').start({allowExpressions: true});
+  require('@google-cloud/trace-agent').start();
+}
+
+const express = require('express');
+const datastore = require('@google-cloud/datastore')();
+const https = require('https');
+const app = express();
+const zlib = require('zlib');
+
+// Error on absolute path values.
+app.get('/:owner/:repo/:tag', (request, response) => {
+  response.set('Access-Control-Allow-Origin', '*');
+  response.status(400).send('Error: Invalid request. Try using a relative path if you are using an absolute path.');
+});
+
+// Redirect requests with incorrectly specified bower_components path.
+app.get('/:owner/:repo/:tag/:before([\\s\\S]*)/bower_components/:after([\\s\\S]*)', (req, response) => {
+  response.set('cache-control', 'max-age=315569000');
+  var url = [req.params.owner, req.params.repo, req.params.tag, req.params.after].join('/');
+  response.redirect(301, url);
+});
+
+app.get('/[^/]+/[^/]+/[^/]+/[^/]+/', (request, response) => {
+  response.sendFile(__dirname + '/inline-demo.html');
+});
+
+app.get('/:owner/:repo/:tag/:name:path(/[\\s\\S]*)', async (req, res) => {
```

Comment:
This is a requirement in Express as .* doesn't work. See note at end of [app.param](https://expressjs.com/en/4x/api.html#app.param).